spec/index.bs

```diff
@@ -1855,6 +1857,11 @@ A <dfn dfn>supported limits</dfn> object has a value for every limit defined by
     <tr class=row-continuation><td colspan=4>
         The maximum value for the arguments of
         {{GPUComputePassEncoder/dispatchWorkgroups(workgroupCountX, workgroupCountY, workgroupCountZ)}}.
+    <tr><td><dfn>maxClipDistancesSize</dfn>
+        <td>{{GPUSize32}} <td>[=limit class/maximum=] <td>0
+        The maximum value for the array size of [=builtin/clip_distances=] in a vertex stage output.
+        When the {{GPUFeatureName/"clip-distances"}} feature is enabled the default value of
+        `maxClipDistancesSize` is 8, otherwise it is always 0.
```

Comment:
Add an entry to "Adapter Capability Guarantees".

With the current PR revision, it would say that `maxClipDistancesSize` must be &ge; 8 if the adapter has the feature, and must be 0 if it does not.